src/Handler/Table/Create/CreateTableHandler.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Table\Create;
+
+use Google\Protobuf\Internal\Message;
+use Keboola\Datatype\Definition\Teradata;
+use Keboola\StorageDriver\Command\Project\CreateProjectCommand;
+use Keboola\StorageDriver\Command\Table\CreateTableCommand;
+use Keboola\StorageDriver\Command\Table\CreateTableCommand\TableColumn;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Shared\Driver\Exception\Exception;
+use Keboola\StorageDriver\Teradata\ConnectionFactory;
+use Keboola\TableBackendUtils\Column\ColumnCollection;
+use Keboola\TableBackendUtils\Column\Teradata\TeradataColumn;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+use Keboola\TableBackendUtils\Table\Teradata\TeradataTableQueryBuilder;
+
+final class CreateTableHandler implements DriverCommandHandlerInterface
+{
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     */
+    public function __invoke(
+        Message $credentials,
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof CreateTableCommand);
+
+        $db = ConnectionFactory::getConnection($credentials);
+
+        // root user is also root database
+        $databaseName = $credentials->getPrincipal();
+        $meta = $command->getMeta();
+        if ($meta !== null) {
+            // override root user and use other database as root
+            $meta = $meta->unpack();
+            assert($meta instanceof CreateProjectCommand\CreateProjectTeradataMeta);
+            $databaseName = $meta->getRootDatabase() === '' ? $databaseName : $meta->getRootDatabase();
+        }
```

Comment:
tohle uplne nevim jestli potrebuju, potom bych musel asi pridat `CreateTableMetadata` object?

Reply:
Hmm :thinking: už vím, sorry. 
Do path by měl jít název databáze, respektive bucket.
Takže si uděláš v tom testu ještě bucket a vytvoříš tabulky v něm. Pak ti ten handler přestane fakčit :)
Stejně tak to bude v drop, path bude mět length 1 a bude tam název bucketu/databáze